wire-ios-utilities/Source/FileManager+Protection.swift

```diff
@@ -38,24 +47,24 @@ extension FileManager {
                 }
                 #endif
 
-                fatal("Failed to create directory: \(url), error: \(error)")
+                throw FileManagerError.failedToCreateDirectory(error)
             }
         }
 
-        // Make sure it's not accessible until first unlock
-        self.setProtectionUntilFirstUserAuthentication(url)
+        // Make sure it's not accessible until first unlock.
+        try setProtectionUntilFirstUserAuthentication(url)
 
-        // Make sure this is not backed up:
+        // Make sure this is not backed up.
         try? url.excludeFromBackup()
```

Comment:
Shouldn't we let this throw now?

```suggestion
        try url.excludeFromBackup()
```

Reply:
yep